docs/orchestration/concepts/api_keys.md

```diff
@@ -199,29 +199,54 @@ mutation {
 
 ## Using API keys with older versions of Prefect
 
-The `prefect auth login` command will not work with API keys and the  `PREFECT__CLOUD__API_KEY` setting will be ignored before version 0.15.0. In older versions, there were authentication tokens. Keys can be used in-place in older versions by setting them in the config or the environment in the `PREFECT__CLOUD__AUTH_TOKEN` setting.
+::: warning
+API tokens are no longer supported as an authentication method.
 
-Using an API key as a token for registering flows
+If you are running a version of Prefect older than 0.15.0, this section describes how you can use API keys for authentication.
+
+If you have logged in with an API key, but a token still exists on your machine, the API key will be used and the token will be ignored.
+:::
+
+If you are running a version of Prefect older than 0.15.0, note that:
+
+- The `prefect auth login` CLI command will not work with API keys.
+- The `PREFECT__CLOUD__API_KEY` setting will be ignored. 
+
+Versions prior to Prefect 0.15.0 used _authentication tokens_. You can use API keys in these older versions of Prefect by setting them in the config or the environment in the `PREFECT__CLOUD__AUTH_TOKEN` setting.
```

Comment:
```suggestion
Versions prior to Prefect 0.15.0 used _authentication tokens_. You can use API keys as drop-in replacements in these older versions of Prefect by setting them in the config or the environment in the `PREFECT__CLOUD__AUTH_TOKEN` setting.
```

Not in love with this suggestion but it might be nice to hint that you'd use them _as_ you would an auth token.

Reply:
Reworded and greatly simplified.